src/runtime/internal/lifecycle.ts

```diff
@@ -11,18 +11,47 @@ export function get_current_component() {
 	return current_component;
 }
 
+/**
+ * Schedules a callback to run immediately before the component is updated after any state change.
+ * 
+ * The first time the callback runs will be before the initial `onMount`
+ * 
+ * https://svelte.dev/docs#run-time-svelte-beforeupdate
+ */
 export function beforeUpdate(fn: () => any) {
 	get_current_component().$$.before_update.push(fn);
 }
 
+/**
+ * The `onMount` function schedules a callback to run as soon as the component has been mounted to the DOM. 
+ * It must be called during the component's initialisation (but doesn't need to live *inside* the component; 
+ * it can be called from an external module).
```

Comment:
```suggestion
 * It can be called from an external module).
```

Reply:
I took the docs from the Svelte documentation site. If this is a mistake, the Svelte docs should be updated too. It does not seem like a mistake though, see https://libguides.royalroads.ca/punctuation-capitalization/capsemicolons#:~:text=When%20using%20a%20semicolon%20to,blue%3B%20the%20birds%20are%20singing.